core/tests/karma-globals.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Globals for the Karma test environment.
+ */
+
+var GLOBALS = {
+  ALLOWED_INTERACTION_CATEGORIES: '',
+  can_edit: true,
+  can_publish: true,
+  CAN_SEND_EMAILS: true,
+  can_translate: true,
+  canDelete: true,
+  canEdit: true,
+  canModifyRoles: true,
+  canReleaseOwnership: true,
+  canUnpublish: true,
+  collectionId: '',
+  collectionTitle: '',
+  CONTINUOUS_COMPUTATIONS_DATA: '',
+  csrf_token: '',
+  DEFAULT_OBJECT_VALUES: null,
+  DEFAULT_TWITTER_SHARE_MESSAGE_DASHBOARD: '',
+  DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR: '',
+  DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER: '',
+  DEMO_COLLECTIONS: '',
+  DEMO_EXPLORATION_IDS: '',
```

Comment:
> This is only to make ts compiler happy - since it searches for all the GLOBALS stuff in karma-globals but that is not used by js. So, I assigned empty values here.

I think we should still use semantically-correct values, otherwise it will confuse developers.

Reply:
Done.